ui/app.js

```diff
@@ -1121,6 +1146,22 @@ $(document).ready(() => {
         $(".close-all").css("filter", "none");
       }
 
+      if (
+        $(".convictions-known-container").css("display") != "none"
+      ) {
+        shouldClose = false;
+        $(".convictions-known-container").fadeOut(250);
+        $(".close-all").css("filter", "none");
+      }
+
+      if (
+        $(".incidents-known-container").css("display") != "none"
+      ) {
+        shouldClose = false;
+        $(".incidents-known-container").fadeOut(250);
+        $(".close-all").css("filter", "none");
+      }
+

```

Comment:
You could extract this code into a function and simply parse the selectors for making it a bit more clean. This would look as follows:

```js
function closeContainer(selector) {
   if (
        $(selector).css("display") != "none"
      ) {
        shouldClose = false;
        $(selector).fadeOut(250);
        $(".close-all").css("filter", "none");
      }
}
```
where you then call the method with the selectors where the current code is:
```js
closeContainer(".convictions-known-container");
closeContainer(".incidents-known-container");
```

Just a small comment - up to you if you want to implement it :)

Reply:
done <3